frontend/src/components/RequestDialog.vue

```diff
@@ -155,15 +169,22 @@ const submitRequestTransaction = async (formResult: {
     targetChainName: formResult.toChainId.label,
     targetAddress: request.targetAddress,
     amount: formResult.amount,
-    fee: feesEther.value,
+    fee: formattedRequestFeeAmount.value, // TODO: Do we need the formatted amount here?!
```

Comment:
I think this value is not used anywhere and could be removed entirely from the type.

Reply:
Nothing simpler than that. 😬

Reply:
✔️